pages/accountLists/[accountListId]/reports/salaryCurrency/[[...contactId]].page.tsx

```diff
@@ -13,79 +12,82 @@ import {
   MultiPageMenu,
   NavTypeEnum,
 } from 'src/components/Shared/MultiPageLayout/MultiPageMenu/MultiPageMenu';
+import {
+  ContactPanelProvider,
+  useContactPanel,
+} from 'src/components/common/ContactPanelProvider/ContactPanelProvider';
 import { FourteenMonthReportCurrencyType } from 'src/graphql/types.generated';
 import { useAccountListId } from 'src/hooks/useAccountListId';
 import { useContactLinks } from 'src/hooks/useContactLinks';
 import useGetAppSettings from 'src/hooks/useGetAppSettings';
-import { getQueryParam } from 'src/utils/queryParam';
-import { ContactsWrapper } from '../../contacts/ContactsWrapper';
 
 const SalaryCurrencyReportPageWrapper = styled(Box)(({ theme }) => ({
   backgroundColor: theme.palette.common.white,
 }));
 
-const SalaryCurrencyReportPage: React.FC = () => {
+const PageContent: React.FC = () => {
```

Comment:
Could you explain in more details, why you needed to edit that coma?

Reply:
Sorry, I'm not sure what you mean. Which comma?